### ISSUE TITLE
tldr-update-docs leads to infinitive loop

### ISSUE THREAD

**Issue by halloleo** [kuanyui/tldr.el#27]
When I issue `tldr-update-docs` (or  ehn I search for the first time for a command via `tldr` emacs gets into an in fibitive download loop. After interrupting (Ctrl-g) I have several *buffers* with names like `tldr-source.zip<X>`

My bufferlist starts like this:

```
 MRL Name                    Size Mode             Filename/Process
 --- ----                    ---- ----             ----------------
[ Default ]
 *   tldr-source.zip<4>   1087249 Fundamental      /tmp/tldr-source.zip
 *   tldr-source.zip<3>   1087249 Fundamental      /tmp/tldr-source.zip
 *   tldr-source.zip<2>   1087249 Fundamental      /tmp/tldr-source.zip
 *   tldr-source.zip      1087249 Fundamental      /tmp/tldr-source.zip
```

Evidently something is going wrong when downloading the page archive. 

When I download the archive by hand and put the `pages subdir in the right place `tldr` works correctly.

**Comment by masasam** [comment]
@halloleo Since the bug of library used by tldr.el has been resolved, it will be resolved after you update MELPA.

**Comment by halloleo** [comment]
I updated all my packages (including the dependency `request`) and I still get an issue: 

tldr still empties the tldr path and then creates this funny buffer:

     *   tldr-source.zip      1064583 Fundamental      /tmp/tldr-source.zip

Now at least it is only *one* buffer, but tldr path is still not filled with the tldr pages.
